crt_portal/cts_forms/templates/forms/index.html

```diff
@@ -43,7 +43,11 @@ <h1>
     {% if data_dict %}
       {% for datum in data_dict %}
         <tr class="tr-status-{{ datum.report.status }}">
-          <td><span class="status-{{ datum.report.status }} row-status-{{ datum.report.status }}">{{ datum.report.status }}</span></td>
+          <td>
+            <span class="status-tag status-{{ datum.report.status }} row-status-{{ datum.report.status }}">
```

Comment:
I think you added an extra `row-status-new` class.

Reply:
The only additional class here is the shared `status-tag`; would you mind taking another look @LindsayYoung?

Reply:
I don't see `row-status-new` defined in the css. I don't think that class was supposed to be in there.

Reply:
Agreed!